packages/discord.js/src/managers/GuildMemberManager.js

```diff
@@ -31,7 +31,7 @@ class GuildMemberManager extends CachedManager {
 
   /**
    * The cache of this Manager
-   * @type {Collection<Snowflake, GuildMember>}
+   * @type {Collection<Snowflake, GuildMember | PartialGuildMember>}
```

Comment:
The whole idea of fixing this was that there *won‘t* be `PartialGuildMember` in the cache. And there is no `PartialGuildMember` in JS land anyway, so this would only make sense in the typings (but is wrong there for the above reason).
Same applies for all other JSDoc comments you added `PartialGuildMember` to